examples/undo-tree/app/undo-tree.js

```diff
@@ -4,27 +4,8 @@ import Pixels    from './stores/pixels'
 export default class UndoTree extends Microcosm {
 
   constructor() {
-    super()
+    super({ maxHistory: Infinity })
```

Comment:
No. It's actually `-Infinity`. By default we only hold on all incomplete transactions. When they finish, we merge them into a cache (See the history plugin for more)